src/index.ts

```diff
@@ -96,16 +104,21 @@ export function mnemonicToEntropy(mnemonic: string, wordlist: string[]): Uint8Ar
  * entropyToMnemonic(ent, wordlist);
  * // 'legal winner thank year wave sausage worth useful legal winner thank yellow'
  */
-export function entropyToMnemonic(entropy: Uint8Array, wordlist: string[]): string {
+export function entropyToMnemonic(entropy: Uint8Array, wordlist: string[]): Uint8Array {
```

Comment:
Maybe keep the method as-is, and create a new one:

```ts
export function entropyToMnemonicIndexes(entropy: Uint8Array, wordlist: string[]): number[] {
  assertEntropy(entropy);
  const words = getCoder(wordlist).encode(entropy);
  return words.map((word) => wordlist.indexOf(word));
}
```

Reply:
Sure we can do this, though we won't be using the old one with this fork...